src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -240,6 +240,7 @@ public DashboardWebApplication(
         builder.Services.TryAddScoped<ComponentTelemetryContextProvider>();
         builder.Services.TryAddSingleton<DashboardTelemetryService>();
         builder.Services.TryAddSingleton<IDashboardTelemetrySender, DashboardTelemetrySender>();
+        builder.Services.AddSingleton<ILoggerProvider, TelemetryLoggerProvider>();
```

Comment:
I'm assuming this usage pattern adds a further `ILoggerProvider` rather than replacing the one that would already have been in place, so you still get normal logging output.

Reply:
@SteveSandersonMS I added a test.